lib/ansible/modules/remote_management/dellemc/idrac/dellemc_idrac_server_config_profile.py

```diff
@@ -0,0 +1,306 @@
+
+RETURN = r'''
+---
+msg:
+  type: str
+  description: Status of the import or export SCP job.
+  returned: always
+  sample: "Successfully imported the Server Configuration Profile"
+scp_status:
+  type: dict
+  description: SCP operation job and progress details from the iDRAC.
+  returned: success
+  sample:
+    {
+      "Id": "JID_XXXXXXXXX",
+      "JobState": "Completed",
+      "JobType": "ImportConfiguration",
+      "Message": "Successfully imported and applied Server Configuration Profile.",
+      "MessageArgs": [],
+      "MessageId": "XXX123",
+      "Name": "Import Configuration",
+      "PercentComplete": 100,
+      "StartTime": "TIME_NOW",
+      "Status": "Success",
+      "TargetSettingsURI": null,
+      "retval": true
+    }
+'''
+
+import os
+from ansible.module_utils.remote_management.dellemc.dellemc_idrac import iDRACConnection
+from ansible.module_utils.basic import AnsibleModule
+try:
+    from omsdk.sdkfile import file_share_manager
+    from omsdk.sdkcreds import UserCredentials
+    from omdrivers.enums.iDRAC.iDRACEnums import (SCPTargetEnum, EndHostPowerStateEnum,
+                                                  ShutdownTypeEnum, ExportFormatEnum, ExportUseEnum)
+except ImportError:
+    pass
+
+
+def run_import_server_config_profile(idrac, module):
+    """Import Server Configuration Profile from a network share."""
+    target = SCPTargetEnum[module.params['scp_components']]
+    job_wait = module.params['job_wait']
+    end_host_power_state = EndHostPowerStateEnum[module.params['end_host_power_state']]
+    shutdown_type = ShutdownTypeEnum[module.params['shutdown_type']]
+    idrac.use_redfish = True
+
+    try:
+        myshare = file_share_manager.create_share_obj(
+            share_path="{0}{1}{2}".format(module.params['share_name'], os.sep, module.params['scp_file']),
+            creds=UserCredentials(module.params['share_user'],
+                                  module.params['share_pwd']), isFolder=False)
+        import_status = idrac.config_mgr.scp_import(myshare,
+                                                    target=target, shutdown_type=shutdown_type,
+                                                    end_host_power_state=end_host_power_state,
+                                                    job_wait=job_wait)
+        if not import_status or import_status.get('Status') != "Success":
+            module.fail_json(msg='Failed to import scp.', scp_status=import_status)
+    except RuntimeError as e:
+        module.fail_json(msg=str(e))
+    return import_status
+
+
+def run_export_server_config_profile(idrac, module):
+    """Export Server Configuration Profile to a network share."""
+    file_format = module.params['export_format'].lower()
+    export_format = ExportFormatEnum[module.params['export_format']]
+    scp_file_name_format = "%ip_%Y%m%d_%H%M%S_scp.{0}".format(file_format)
+    target = SCPTargetEnum[module.params['scp_components']]
+    job_wait = module.params['job_wait']
```

Comment:
can you pass `module.params['job_wait']` directly without using this temp variable